### PR TITLE
[master<changelog-lab-260] Lab 2.6.0 changelog

### PR DESCRIPTION
**Pull request by vpavicic**
### Description

Added changelog from 
https://www.notion.so/memgraph/Lab-v2-6-0-Multiple-query-results-a23fd3634e1040bb89c643bdf6719846

### CODE REVIEW

**Comment by vpavicic** [comment]
I need to add links + gifs... will do later today / Monday

File: lab/changelog.md [hunk 1]
```diff
@@ -4,9 +4,53 @@ title: Changelog
...
+  longer becomes unresponsive. 
+- Using a new line character `\n` in the query module doesn’t result in a new
+  line, but an explicit character `\n`.
+- Viewing the code of multiple query modules in the split screen no longer now
+  works as expected.
+- Notifications no longer mix with the Query Editor and Query Collections visual
+  elements.
+- Using shortcut keys CMD/CTRL + S to save a query to a query collection now
+  works for queries within a single query execution. In a query collection, the
+  shortcut will initiate a manual save of the query within the query collection.
```
Comment (vpavicic) [inline lab/changelog.md:42]: I don't really understand this one

Comment (vpavicic) [inline lab/changelog.md:42]:
```suggestion
- Pressing CMD/CTRL + S will save a query within a query collection execution section as intended.
```

File: lab/changelog.md [hunk 2]
```diff
@@ -4,9 +4,53 @@ title: Changelog
...
+
+- The initial node count has been removed from the connection initialization so
+  connecting to the Memgraph instance containing a huge number of nodes will no
+  longer cause a timeout. 
+- Run History now logs queries as expected.
+- When switching between the map view and the default view the graph view no
+  longer becomes unresponsive. 
+- Using a new line character `\n` in the query module doesn’t result in a new
+  line, but an explicit character `\n`.
+- Viewing the code of multiple query modules in the split screen no longer now
```
Comment (tonilastre) [inline lab/changelog.md:36]:
```suggestion
- Viewing the code of multiple query modules in the split screen now
```

**Pull request merged**